docs/modules/ROOT/pages/exoscale-dbaas/redis/index.adoc

```diff
@@ -0,0 +1,20 @@
+= Exoscale DBaaS - Redis
+
+Welcome to the https://www.exoscale.com/dbaas/redis/[Exoscale DBaaS Managed Redis^] service in the VSHN Application Catalog.
+
+Please find more about the service description including pricing in our https://products.docs.vshn.ch/products/appcat/exoscale_dbaas.html[product documentation site].
+
+TIP: Get started with xref:exoscale-dbaas/redis/create.adoc[Creating a Redis instance].
+
+== Notes
+
+Some important notes about this service:
+
+* Connections are only possible using authenticated TLS encrypted connections.
```

Comment:
Redis does not expose a certificate upon creation.

Reply:
Ah true, I should add a few words about the ca.crt secret and on Redis that it's Let's Encrypt. Will add that...

Reply:
Also FYI we were able to connect to all services without the ca certificate

Reply:
> Also FYI we were able to connect to all services without the ca certificate.

How do you mean that? IMHO that depends on the application if a verification of the self-signed certificate is made or not.

Reply:
Never mind my last comment, I think I got smth wrong.